src/api/update.ts

```diff
@@ -5,6 +5,17 @@ import { read } from "./read"
 import { updateTags } from '../updateTags'
 import { write, WriteCallback } from "./write"
 
+/**
```

Comment:
There is no logic, but TypeScript is unable to infer the callback parameter types otherwise.
Before, it was not possible to call `NodeID3.update({}, buffer, (err, data) => {})` because err/data would be any.

I don't know why changing the order fixes it :/ Do you have any idea?

Reply:
I did a test and I do not observe this behaviour this is strange. Let's experiment offline.